tests/core_test.go

```diff
@@ -24,6 +27,17 @@ var _ = Describe("Core API endpoint tests", func() {
 		Expect(err).ToNot(HaveOccurred())
 	})
 
+	AfterEach(func() {
+		for _, handler := range cleanupHandlers {
+			err := handler()
+			if err != nil {
+				_, _ = fmt.Fprintf(GinkgoWriter, "error when cleaning up tests: %s", err.Error())
+			}
+		}
+
+		cleanupHandlers = []CleanUpHandler{}
+	})
+
 	Context("Location endpoint", func() {
```

Comment:
this is not a `Context` but a `Describe`, but am fine with this staying .. but might clash with the suggested Context below (not sure rn if they can be nested)